### ISSUE TITLE
Release 1.2.1 and1.2.0 missing build tools (see screenshot) since 1.1.0. 

### ISSUE THREAD

**Issue by shurushetr** [rockowitz/ddcutil#239]
Hi, not sure if this on purpose, not critical, I will try to compile with files from 1.1.0 tar.
Building fo Rasp. PI4, current release in buster repo available is 0.9.2 at least for me
![2021-12-06 14_20_53-C__Users_User_AppData_Local_Temp_ddcutil-1 1 0 tar gz_ddcutil-1 1 0 tar_ddcutil-](https://user-images.githubusercontent.com/765095/144909064-54c49286-939f-4b56-bdcb-22f864c557d7.png)
.

**Comment by shurushetr** [comment]
Looks like tar files from git don't have the tools but this [http://www.ddcutil.com/tarballs/ddcutil-1.2.1.tar.gz](http://www.ddcutil.com/tarballs/ddcutil-1.2.1.tar.gz) has.

**Comment by rockowitz** [comment]
I'm not sure what you mean by "tools", but I see that the config directory of autotools generated files is empty on github but is included in the website tarball.  The web site tarball is created by autotools command "make diff", and contains some generated files.  Either tarball works if the build process begins with command "autoreconf".  However, the divergences between these tarballs should be minimized and documented. Thanks for pointing out the situation.  Is the difference just a curiosity for you, or has it caused build problems?

**Comment by shurushetr** [comment]
I figured it out, I guess my limited knowledge of git is in play here. I
was trying to build the latest version of the app by compiling from source,
and git release archive didn't have anything in build folder. I then went
on the website and there was a previous version of tarball (110 I think)
with build folder complete. So I just replaced version numbers in URL for
older tarball and got correct tarball from website.
